docs/pages/guides/using-hermes.mdx

```diff
@@ -108,6 +108,6 @@ Alternatively, you can use the JavaScript inspector from the following tools:
 
 ### Can I use Remote Debugging with Hermes?
 
-One of the many limitations of [remote debugging](/more/glossary-of-terms/#remote-debugging) is that it does not work with modules built on top of [JSI](https://github.com/react-native-community/discussions-and-proposals/issues/91). For example, if your app uses [react-native-reanimated](https://github.com/software-mansion/react-native-reanimated) version 2 or higher, [remote debugging will not work](https://docs.swmansion.com/react-native-reanimated/docs/#known-problems-and-limitations).
+One of the many limitations of [remote debugging](/more/glossary-of-terms/#remote-debugging) is that it does not work with modules built on top of [JSI](https://github.com/react-native-community/discussions-and-proposals/issues/91), such as [react-native-reanimated](https://github.com/software-mansion/react-native-reanimated) version 2 or higher.
```

Comment:
Do you think its appropriate here to link to this Reanimated docs link: https://docs.swmansion.com/react-native-reanimated/docs/guide/debugging/?

Reply:
I dunno, it's a good document, but Reanimated is mostly just serving as an example here, it's not really about debugging reanimated, per se.

Reply:
Got it!